history-of-quakers/modernized/16-chapter-14.adoc

```diff
@@ -99,10 +110,13 @@ for they seemed to imagine that the progress of that doctrine,
 which he so powerfully and successfully had preached,
 by his decease would have been stopped or retarded?
 but they made a wrong reckoning.
-Francis Howgill then gave forth a kind of epicedium
+Francis Howgill then gave forth a kind of _epicedium_
 (i.e. lamentation) which though in prose,
 yet was not void of poetical expressions, and was as follows.
 
+[.embedded-content-document.letter]
```

Comment:
Not sure if this should be labeled a letter or not.

Reply:
yeah, i think it would be slightly better as a testimony, although that's not perfectly clear from the context. i had one other unrelated thing I wanted to change in this file, so I actually went ahead and changed this, so no need for you to do anything.